ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/io/FileOperations.java

```diff
@@ -136,18 +137,41 @@ private void copyMmPath() throws LoginException, IOException {
     }
   }
 
-
-
   /**
    * This needs the root data directory to which the data needs to be exported to.
    * The data export here is a list of files either in table/partition that are written to the _files
    * in the exportRootDataDir provided.
    */
   private void exportFilesAsList() throws SemanticException, IOException {
-    // This is only called for replication that handles MM tables; no need for mmCtx.
-    try (BufferedWriter writer = writer()) {
-      for (Path dataPath : dataPathList) {
-        writeFilesList(listFilesInDir(dataPath), writer, AcidUtils.getAcidSubDir(dataPath));
+    if (dataPathList.isEmpty()) {
```

Comment:
Even if dataPathList is empty, need to create empty file. I think, REPL LOAD will expect it. At least that's what base code does.